package.json

```
@@ -39,8 +43,5 @@
 		"unplugin-isolated-decl": "^0.6.5",
 		"vitest": "^2.0.5"
```

https://github.com/sveltejs/cli/security/dependabot shows some issues from vite. Perhaps we could upgrade it to address those. I didn't want to do it while you were working on testing stuff, but if it doesn't break things, you might want to try doing it here
```suggestion
		"vitest": "^2.1.4"
```